### DISCUSSION TITLE
hagelslag

### DISCUSSION THREAD

**Discussion by giscus** [Roald87/roaldin.ch#15] [Announcements]
# hagelslag

Naast pindakaas eet ik ook graag hagelslag. Aanvankelijk was hagelslag hier moeilijk te vinden en namen we het altijd mee uit Nederland. Echter, sinds we in Wettingen wonen, heeft onze grote supermarkt één soort hagelslag dat ook een van mijn favorieten is: de Ruyter puur. De verpakking is iets anders dan de Nederlandse. Op de voorkant staat wel een boterham met hagelslag, maar op de achterkant staan serveersuggesties. Je ziet voorbeelden van hagelslag op cake, in de yoghurt of op ijs. Ik heb mensen als bedankje ook wel eens een Nederlandse hagelslagverpakking gegeven. Zij doen het dan door de yoghurt of op ijs. De brood-optie moet ik ze altijd uitleggen.

https://roaldin.ch/hagelslag

**Comment by oom-wim** [comment]
Heeft broodbelegkeuze een erfelijkheidsfactor?

**Reply by Roald87** [reply]
Vermoedelijk niet als ik kijk naar de kaasconsumptie van mijn brongenen.
